ideascube/configuration/registry.py

```diff
@@ -22,6 +22,12 @@
             'default': getattr(settings, 'IDEASCUBE_NAME', 'Ideas Cube'),
         },
     },
+    'ideascube': {
```

Comment:
All config options are ideascube configs. Maybe `home` would be a better namespace?

Also, I prefer my bikeshed painted black!